coreneuron/nrnoc/mech_mapping.cpp

```diff
@@ -0,0 +1,58 @@
+#include "coreneuron/nrnoc/mech_mapping.hpp"
+#include "coreneuron/utils/data_layout.hpp"
+#include "coreneuron/nrnoc/nrnoc_ml.h"                    // for Memb_list definition
+#include <iostream>
+#include <cstring>
+#include <map>
+
```

Comment:
for consistency include standard headers at the top

Reply:
I think to ensure that all files are correctly self-contained, it is better to include standards and system library headers right after local headers. it also goes toward making coreneuron modular ( coreneuron as a library)